client/verta/verta/endpoint/build/_build.py

```diff
@@ -89,10 +118,36 @@ def date_created(self) -> datetime:
     def status(self) -> str:
         return self._json["status"]
 
+    @property
+    def location(self) -> Optional[str]:
+        location = self._json.get("location")
+        if location is None:
+            location = self.json.get("creator_request", dict()).get("external_location")
+        return location
+
+    @property
+    def requires_root(self) -> Optional[bool]:
+        return self._json.get("requires_root")
+
+    @property
+    def scan_external(self) -> Optional[bool]:
+        return self._json.get("scan_external")
+
+    @property
+    def self_contained(self) -> Optional[bool]:
+        return self._json.get("self_contained")
+
     @property
     def message(self) -> str:
         return self._json.get("message") or self._EMPTY_MESSAGE
 
+    @message.setter
+    def message(self, message: str) -> None:
```

Comment:
I would discourage the use of `property` `setter`s altogether. It's way too easy for a user to accidentally type

```python
build.messge = "foo"
```

(note "messge" is misspelled) and Python wouldn't raise an exception.

`set_message()` would be preferable.